tutorials/rendering/jitter_stutter.rst

```diff
@@ -147,6 +147,224 @@ iOS
+
+Bad HDMI / DisplayPort cable
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A broken cable may appear to function properly, but cause signal synchronization
+issues that go apparently unnoticed. However these issues can manifest when V-Sync
+is on and apps (including, but not limited, to Godot) running in exclusive fullscreen
+constantly have FPS jitter or slowdowns even in the most basic of scenes.
+
+If you see simple demo apps struggle to reach 60 fps (e.g. it reaches 60 fps then
+every 2 seconds slows down to 55 or 40 FPS and then goes back up) you may have a bad cable.
+
+**Solution:** Replace the HDMI / DP cable.
+
+**Workaround:** Disable V-Sync and use the FPS limiter by launching with: ``--disable-vsync --max-fps <fps>``
+
+Broken Monitor Firmware
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Similar to a bad HDMI / DP cable, a poor interaction between your monitor firmware and
+your GPU can cause jitter or periodic slowdown.
+
+Unfortunately the simple answer is to replace the monitor. You might have luck contacting
+your Monitor manufacturer (e.g. LG, Samsung, Viewsonic, etc) and/or your GPU vendor
+(e.g. NVIDIA, AMD, Intel) and ask them for a solution or workaround.
+
+**Solution:** Replace the monitor, or ask the manufacturer for a firmware/driver update.
+
+**Workaround:** Disable V-Sync and use the FPS limiter by launching with: ``--disable-vsync --max-fps <fps>``
+
+Multiple Monitors
+~~~~~~~~~~~~~~~~~
+
+If you've got multiple monitors, ensure they are all in the same resolution, frequency (Hz),
+and bit depth (e.g. 16-bit vs 24-bit, HDR vs SDR).
+
+If they're not, change their settings until they match.
+
+If problems persist, try disabling all but one of the monitors.
+
+Overheating and throttling
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Jitter or stutter may be caused by an overheating CPU and/or GPU that throttles itself down.
+You can use temperature monitor software to see if this is the case.
+
+**Solution:** Fix the cooling problem.
+
+Inconsistent Power Saving
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In some cases the OS or driver keeps switching the CPU and/or GPU frequencies up and
+down; causing an uneven experience.
+
+**Solution (Windows):** Set the Power Profile to "High Performance":
+
+.. image:: img/windows-high-performance-profile.webp
+
+.. tip::
+
+    Sometimes "CPU Boost" where a single CPU core goes into overdrive can be a problem.
+    You can prevent CPU Boost by editing your *current* energy plan and set
+    **Processor power management > Maximum processor state** to 99% instead of 100%:
+
+    .. image:: img/windows-no-boost.webp
+
+For NVIDIA on Windows, set "Power management mode" on "Prefer maximum performance".
+
+.. image:: img/nv-power-profile.webp
+
+**Solution (Linux):** Set the CPU governor to Performance:
+
+.. code:: sh
+
+    sudo cpupower frequency-set -g performance
+
+    # AMD
+    echo high | sudo tee /sys/class/drm/card1/device/power_dpm_force_performance_level
```

Comment:
For future reference, this should be less of a necessity on a Linux distribution with a recent Mesa stack since https://gitlab.freedesktop.org/drm/amd/-/issues/1500 has been resolved. However, it can still improve performance (and latency) at times, so it's worth mentioning.